src/interface/interface.json

```diff
@@ -1590,6 +1590,57 @@
             "body": "",
             "headerColor": ""
           }
+      },    
+      "Table": {
```

Comment:
sound good. Yes will take a look. Also the gemini snapshot is the first file in this PR. It is checked in. I will address the other comments.